cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,82 @@
 /// <reference types='cypress' />
+Cypress.on('uncaught:exception', (_err) => {
+  return false;
+});
+
+let user;
 
 describe('Web Tables page', () => {
-  it('', () => {
+  before(() => {
+    // eslint-disable-next-line arrow-parens
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('Should check pagination.', () => {
+    cy.get('.-previous > .-btn').should('contain', 'Previous');
+    cy.get('.-next > .-btn').should('contain', 'Next');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+  });
+
+  it('Rows count selection.', () => {
+    cy.get('select').select('10 rows');
+  });
+
+  it('Create a new user.', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+  });
+
+  it('Delete user.', () => {
+    cy.get('#delete-record-1').click();
```

Comment:
where are your assertions? how do you know that the user was deleted? you need to add an assertion .should() to check the state of the app after any actions

Reply:
Changes in commit [c24a87a](https://github.com/mate-academy/qa_cypress_e2e_web_tables/pull/188/commits/c24a87a0c653936f8c3e3a480f55479253281410)